config/engines.yml

```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
+shellcheck:
+  image: codeclimate/codeclimate-shellcheck
+  description: A static analysis tool for shell scripts
```

Comment:
Good point. Other descriptions in the file are currently fragments, so for now I think just adding the period is fine, and we can wait for @scriberty to decide it's bad & needs changing.